pkgs/by-name/em/emacsclient-commands/package.nix

```
@@ -0,0 +1,28 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, buildGoModule, runCommandLocal }:
```

while changing the formatting you can also remove
```suggestion
{ lib, fetchFromGitHub, buildGoModule }:
```
since they are unused. And we should avoid adding pkgs to the scope of this package and instead hook in the buildInputs like gnumake directly.